torch/onnx/symbolic_opset12.py

```diff
@@ -481,3 +484,26 @@ def tensordot(g: jit_utils.GraphContext, input_a, input_b, dims_a, dims_b, out=N
 
     shape_sizes = [left_sizes_a, left_sizes_b]
     return opset9._reshape_from_tensor(g, output, shape_sizes)
+
+
+@_onnx_symbolic("aten::logit")
+@_beartype.beartype
+def logit(g: jit_utils.GraphContext, self: torch._C.Value, eps: torch._C.Value):
+    one = g.op("Constant", value_t=torch.tensor(1.0))
+
+    if not symbolic_helper._is_none(eps):
+        eps = g.op(
+            "Cast", eps, to_i=_type_utils.JitScalarType.from_value(self).onnx_type()
+        )
+        one_sub_eps = g.op("Sub", one, eps)
+        self_less_equal_one_sub_eps = g.op("LessOrEqual", self, one_sub_eps)
```

Comment:
very small nit: we could do for opset<12 with 'Not' + 'Greater' to replace 'LessOrEqual' right? (Or even better, just use 'Greater', and swap the `self` and `other` inputs?)

Reply:
Moved the whole thing to symbolic_opset9